.gitignore

```diff
@@ -1 +1,2 @@
 code/target/
+code/src/main/webapp/WEB-INF/credentials.json
```

Comment:
High level comment. 

This is at least 9 different PRs you could have easily split up:

1) Updating the abstract methods signatures and the code that implements them
2) Updating the logic to updateUser and getPayload
3) The GetCalendarFor*Users() stubs
4) GetNegativeUserInfoServlet
5) GetPeopleDataForPositiveUser
6) Changing the error handling case for some reason
7) GetPositiveUserInfoServlet
8) NegativeUser fixes
9) PositiveUser fixes

That along with much of this code not being complete at the time you're asking for review. It's really advantageous to splitting this up because we could approve different simpler things seperately and quickly, leaving the harder chunks of code for discussion.

Reply:
So, talked with Nico off PR. 

His comments were that it was designed to be large so because "the goal is to set up all the servlets needed for APIs and once the branch is merge we can make PRs that only alter one file (a runnable)."

To which I say, a huge portion of this is unrelated to this still. And we could have also already merged a very large portion of this code because the sticking points are only on a small subset of the files. For instance, the runnable stubs could have already been pushed. The Positive and Negative user fixes are both very minor fixes that could have been handled separately (and are bug fixes anyway that aren't in files that are otherwise modified). 

It honestly makes the entire process faster to have smaller PRs. I think the concern is we have to get it out, so we're doing it in one big PR to get it done. But this really does actually SLOW things down. 

Most of this code could have been merged already if it was split off into its own PR. Everything else is now held up by the discussion on these last parts.

Reply:
I guess I was referring mostly to the files related to this PR. But yes you are right, this process is drawn out by all the files that vaguely relate to this branch's goal. I removed those files and will make a new PR for them.